products/compute/terraform.yaml

```diff
@@ -138,8 +138,42 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       name: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           regex: '^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$'
+      cdnPolicy: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+  BackendBucketSignedUrlKey: !ruby/object:Overrides::Terraform::ResourceOverride
+    exclude_import: true
+    mutex: signedUrlKey/{{project}}/backendBuckets/{{backend_bucket}}/
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "backend_bucket_signed_url_key"
+        primary_resource_id: "backend_key"
+        vars:
+          key_name: "test-key"
+          backend_name: "test-signed-backend-bucket"
+          bucket_name: "test-storage-bucket"
+        skip_test: true
+    properties:
+      backendBucket: !ruby/object:Overrides::Terraform::PropertyOverride
+        ignore_read: true
+      name: !ruby/object:Overrides::Terraform::PropertyOverride
+        validation: !ruby/object:Provider::Terraform::Validation
+          regex: '^(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)$'
+        ignore_read: true
+      keyValue: !ruby/object:Overrides::Terraform::PropertyOverride
+        sensitive: true
+        ignore_read: true
+    docs: !ruby/object:Provider::Terraform::Docs
+      warning: |
+        All arguments including the key's value will be stored in the raw
+        state as plain-text. [Read more about sensitive data in state](/docs/state/sensitive-data.html).
+        Because the API does not return the sensitive key value,
+        we cannot confirm or reverse changes to a key outside of Terraform.
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      decoder: templates/terraform/decoders/backend_signed_url_key.go.erb
   BackendService: !ruby/object:Overrides::Terraform::ResourceOverride
     exclude: true
+  BackendServiceSignedUrlKey: !ruby/object:Overrides::Terraform::ResourceOverride
```

Comment:
Will this be broken out to a future PR? Should we break out the `api.yaml` as well in that case?

Reply:
I'm breaking it out in a separate PR but I don't feel like splitting the inspec/ansible code changes  is useful, so I'd prefer to just exclude here and include it back in later

Reply:
particularly because we autogen backend bucket but not service, and the other providers have them autogen'd

Reply:
Fair enough! My thinking here is just that submitting the api.yaml + terraform.yaml at the same time for BackendServiceSignedUrlKey will be a little more sensible history

Also, *hopefully* that won't be true for long now that we support convenience fields 🙂